winapi/jobobj.go

```diff
@@ -35,13 +35,54 @@ const (
 	JOB_OBJECT_MSG_ABNORMAL_EXIT_PROCESS = 8
 	JOB_OBJECT_MSG_PROCESS_MEMORY_LIMIT  = 9
 	JOB_OBJECT_MSG_JOB_MEMORY_LIMIT      = 10
+
+	JOB_OBJECT_LIMIT_KILL_ON_JOB_CLOSE          = 0x2000
+	JOB_OBJECT_LIMIT_DIE_ON_UNHANDLED_EXCEPTION = 0x400
+	JOB_OBJECT_LIMIT_ACTIVE_PROCESS             = 8
+	JOB_OBJECT_LIMIT_JOB_MEMORY                 = 0x200
+	JOB_OBJECT_LIMIT_JOB_TIME                   = 4
+	JOB_OBJECT_LIMIT_PROCESS_MEMORY             = 0x100
+	JOB_OBJECT_LIMIT_PROCESS_TIME               = 2
+	JOB_OBJECT_LIMIT_WORKINGSET                 = 1
+	JOB_OBJECT_LIMIT_AFFINITY                   = 0x00000010
 )
 
 type JOBOBJECT_ASSOCIATE_COMPLETION_PORT struct {
 	CompletionKey  uintptr
 	CompletionPort syscall.Handle
 }
 
+type IO_COUNTERS struct {
+	ReadOperationCount  uint64
+	ReadOperationCount  uint64
+	WriteOperationCount uint64
+	OtherOperationCount uint64
+	ReadTransferCount   uint64
+	WriteTransferCount  uint64
+	OtherTransferCount  uint64
+}
+
+type JOBOBJECT_BASIC_LIMIT_INFORMATION struct {
+	PerProcessUserTimeLimit uint64
```

Comment:
s/uint64/int64/

No?

PerProcessUserTimeLimit is a LARGE_INTEGER. But LARGE_INTEGER is "A 64-bit signed integer. The range is –9223372036854775808 through 9223372036854775807 decimal." according to https://msdn.microsoft.com/en-us/library/windows/desktop/aa383751(v=vs.85).aspx